### PR TITLE
Updated pydantic to version 1.

### PR DESCRIPTION
**Pull request by jaheba**
Related to #456.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
-matplotlib>=3.0,<4
-numpy>=1.14,<2
+matplotlib~=3.0
+numpy~=1.14
 pandas>=0.25,<0.26
-pydantic>=0.28,<2
-tqdm>=4.23,<5
-ujson>=1.3,<2
+pydantic~=1.1
+tqdm~=4.23,<5
```
Comment (lostella) [inline requirements/requirements.txt]:
```suggestion
tqdm~=4.23
```

File: requirements/requirements.txt [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
+ujson~=1.35,<2
```
Comment (lostella) [inline requirements/requirements.txt]:
```suggestion
ujson~=1.35
```

File: requirements/requirements.txt (hunk 1 above)
Comment (lostella) [reply requirements/requirements.txt]: (aren't these equivalent?)

**Comment by lostella** [comment]
The following needs to go in `gluonts.core.component`, lines 344-346, in order to get rid of the doctest complaint:
```python
    pydantic.error_wrappers.ValidationError: 1 validation error for ComplexNumberModel
    y
      none is not an allowed value (type=type_error.none.not_allowed)
```

File: requirements/requirements.txt (hunk 1 above)
Comment (jaheba) [reply requirements/requirements.txt]: Yes, but IMO `~=` looks a bit nicer.

**Pull request merged**